app/src/main/java/com/hedvig/app/SplashActivity.kt

```diff
@@ -11,24 +14,43 @@ import com.hedvig.app.feature.profile.ui.payment.TrustlyActivity
 import com.hedvig.app.feature.referrals.ReferralsReceiverActivity
 import com.hedvig.app.service.LoginStatus
 import com.hedvig.app.service.LoginStatusService
+import com.hedvig.app.util.extensions.observe
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.rxkotlin.plusAssign
 import io.reactivex.schedulers.Schedulers
 import org.koin.android.ext.android.inject
+import org.koin.android.viewmodel.ext.android.viewModel
 import timber.log.Timber
-import java.util.Locale
+import java.util.*
+
 
 class SplashActivity : BaseActivity() {
     private val loggedInService: LoginStatusService by inject()
+    private val birthdayViewModel: BirthdayViewModel by viewModel()
+
+    private var isBirthday: Boolean = false
 
     override fun onStart() {
         super.onStart()
 
+        birthdayViewModel.apply {
```

Comment:
If we move the `getBirthDay()`-call to an `init`-block in the ViewModel, we can skip using `apply` here and simply observe directly on the `isBirthdayData`-property.